src/AppBundle/Entity/Answer.php

```diff
@@ -0,0 +1,124 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: lukas
+ * Date: 16.11.13
+ * Time: 22.30
+ */
+
+namespace AppBundle\Entity;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity()
+ * @ORM\Table(name="answers")
+ */
+class Answer
+{
+    /**
+     * @ORM\Column(type="integer")
+     * @ORM\GeneratedValue(strategy="AUTO")
+     */
+    private $id;
+    /**
+     * @ORM\Column(type="integer")
+     * @ORM\ManyToOne(targetEntity="questions")
+     */
+    private $questionId;
+    /**
```

Comment:
While working with entities, you tipically will lazy load whole object, and then questionId name will not represent quostion object. Better practice is to call it question